test/integration/integration_test.go

```diff
@@ -55,10 +55,13 @@ func TestMain(m *testing.M) {
 	if hostFQDN == "unknown" {
 		fmt.Printf("cannot get FQDN")
 	}
+	nodeIP := os.Getenv("X_CSI_NODE_IP")
 	if os.Getenv("X_CSI_CUSTOM_TOPOLOGY_ENABLED") == "true" {
-		nodeName := host + nodeIDSeparator + hostFQDN + nodeIDSeparator + os.Getenv("X_CSI_NODE_IP")
+		nodeName := host + nodeIDSeparator + hostFQDN + nodeIDSeparator + nodeIP
 		os.Setenv("X_CSI_NODE_NAME", nodeName)
 	}
+	nodeNameWithoutFQDN := host + nodeIDSeparator + nodeIP + nodeIDSeparator + nodeIP
+	os.Setenv("X_CSI_NODE_NAME_NO_FQDN", nodeNameWithoutFQDN)
```

Comment:
Where are we using this X_CSI_NODE_NAME_NO_FQDN, there is no environment variable added with this name

Reply:
this is created only for UT purpose, not being used elsewhere.